{"uuid": "8f058f60-b823-4023-abaa-4385ccef2235", "children": ["5122361e-fc20-4468-8f72-dc47022f1563", "2ae92020-72c9-482d-bc6b-7d1f90beedda", "559733dd-efae-47f7-ac64-9988fee2ad50", "56eee082-48eb-4014-9b53-0685375ced73"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "AttributeError: cls not available in session-scoped context\n", "trace": "  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/tests/vwo_login/conftest.py\", line 18, in setup\n    request.cls.driver = driver\n    ^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 495, in cls\n    raise AttributeError(f\"cls not available in {self.scope}-scoped context\")\n"}, "start": 1693480529135, "stop": 1693480560238}], "afters": [{"name": "setup::fin", "status": "passed", "start": 1693480560293, "stop": 1693480560419}], "start": 1693480529135, "stop": 1693480560419}